pkgs/tools/games/opentracker/default.nix

```diff
@@ -1,38 +1,43 @@
-{ lib
-, stdenv
-, buildDotnetModule
-, fetchFromGitHub
-, autoPatchelfHook
-, wrapGAppsHook
-, dotnetCorePackages
-, fontconfig
-, gtk3
-, openssl
-, libX11
-, libXi
-, xinput
+{
+  lib,
+  stdenv,
+  buildDotnetModule,
+  fetchFromGitHub,
+  autoPatchelfHook,
+  wrapGAppsHook,
+  dotnetCorePackages,
+  fontconfig,
+  gtk3,
+  libunwind,
+  openssl,
+  xinput,
+  xorg,
 }:
-
 buildDotnetModule rec {
   pname = "opentracker";
-  version = "1.8.2";
+  version = "1.8.5";
 
   src = fetchFromGitHub {
     owner = "trippsc2";
     repo = pname;
-    rev = version;
-    sha256 = "0nsmyb1wd86465iri9jxl3jp74gxkscvnmr3687ddbia3dv4fz0z";
+    rev = "refs/tags/${version}";
+    hash = "sha512-nWkPgVYdnBJibyJRdLPe3O3RioDPbzumSritRejmr4CeiPb7aUTON7HjivcV/GKor1guEYu+TJ+QxYrqO/eppg==";
   };
 
-  dotnet-runtime = dotnetCorePackages.runtime_3_1;
-  dotnet-sdk = dotnetCorePackages.sdk_3_1;
+  patches = [./remove-project.patch];
```

Comment:
I feel like running `dotnet sln` in `prePatch` is going to be easier to understand and less likely to break

Reply:
`dotnet sln remove` doesn't work with project GUIDs, only with file paths. The file path doesn't exist anymore in the repo, so `dotnet sln` can't remove it.